x-pack/legacy/plugins/canvas/public/lib/run_interpreter.js

```diff
@@ -33,7 +36,7 @@ export function runInterpreter(ast, context = null, options = {}) {
         });
       }
 
-      return new Error(`Ack! I don't know how to render a '${getType(renderable)}'`);
+      return new Error(strings.getRenderErrorMessage(getType(renderable)));
```

Comment:
Do we want to translate thrown errors like this that aren't displayed to the user? (Or does this somehow get displayed to the user?)

Reply:
Yeah this is a lil weird. I thought the error would get picked up later down the line but it looks like it doesn't. Pretty funky. I backed out of my changes here